CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ All notable changes for each version of this project will be documented in this
     - Added `horizontalSummariesPosition` property to the `pivotUI`, configuring horizontal summaries position.
     - Keyboard navigation now can move in to row headers back and forth from any row dimension headers or column headers.
     - Added keyboard interactions for row dimension collapse using `Alt + Arrows` and row headers sorting using `Ctrl + Arrow Up/Down`.
+- `IgxIcon`, `IgxIconService`
+    - You can now register icons by reference via the `IgxIconService`. To learn more check out the [documentation](https://www.infragistics.com/products/ignite-ui-angular/angular/components/icon-service).
```

Comment:
I guess somewhere in here we should mention that in order to facilitate the new functionality, the existing `registerFamilyAlias` has been deprecated in favor of the new `setFamily` that has more options.

Reply:
Also in the API doc on `registerFamilyAlias` can you update it with the version to match the expected format:
```
@deprecated in version 18.1.0. Use `setFamily` instead.
```